tasks.rb

```diff
@@ -1897,6 +1898,9 @@ def set_measure_argument_values(hpxml_file, args, sch_args, orig_parent)
   elsif ['base-hvac-seasons.xml'].include? hpxml_file
     args['hvac_control_heating_season_period'] = 'Nov 1 - May 1'
     args['hvac_control_cooling_season_period'] = 'Jun 1 - Oct 1'
+  elsif ['base-hvac-seasons-auto.xml'].include? hpxml_file
+    args['hvac_control_heating_season_period'] = HPXML::BuildingAmerica
+    args['hvac_control_cooling_season_period'] = HPXML::BuildingAmerica
```

Comment:
This doesn't seem like a particularly useful sample file since the HPXML ends up with hardcoded seasons in it. So it's no different than the base-hvac-seasons.xml sample file. I suggest removing it.

If your reason for doing this is to test that the BuildResHPXML measure arguments work, that should be a unit test like I suggested above. And it has the bonus of checking for the appropriate expected values.